ui/package.json

```diff
@@ -73,6 +73,7 @@
     "gzip-size": "4.1.0",
     "jquery": "3.2.1",
     "lodash": "4.17.4",
+    "npm": "^6.0.1",
```

Comment:
why is this added?

Reply:
hmm.. I ran `npm i npm` and it added itself. I'll take it out.